1_two_sum.py

```diff
@@ -0,0 +1,11 @@
+from typing import List
+
+
+class Solution:
+    def twoSum(self, nums: List[int], target: int) -> List[int]:
+        lookup = {}
```

Comment:
You should name the dict variable "from_xxx_to_yyy", so in this case it's better to use "from_num_to_supplement" for example.